Makefile

```diff
@@ -49,6 +49,7 @@ out/delta.tar.gz: bin/init bin/vsockexec bin/cmd/gcs bin/cmd/gcstools bin/cmd/ho
 	cp bin/cmd/gcs rootfs/bin/
 	cp bin/cmd/gcstools rootfs/bin/
 	cp bin/cmd/hooks/wait-paths rootfs/bin/
+	cp bin/internal/tools/snp-report rootfs/bin/
```

Comment:
Is this just a debug tool? Could we leave it out of the rootfs and just mount it in when needed?

Reply:
I added `INCLUDE_UTILITIES` parameter and copy the binary only if it's set to `1`. I think it'll be useful when producing a test package.